doto-frontend/src/components/pages/Calendar/Calendar.js

```diff
@@ -62,7 +63,7 @@ const Calendar = () => {
         };
         fetchUserInfo();
         fetchTasks();
-    }, []);
+    });
```

Comment:
This is not ideal, will cause a call on each re render (see comments on #238). Can you change it to [setTheme], cheers.

Reply:
Made the change, thanks for explaining that!